templates/master/01-master-container-runtime/_base/files/etc-subgid.yaml

```diff
@@ -0,0 +1,5 @@
+mode: 0644
+path: "/etc/subgid"
+contents:
+  inline: |
+    containers:100000:65536
```

Comment:
65536 is a very small range :-)

Reply:
per-container? I just chose the default. do you have a different recommendation?

Reply:
that is for the entire `containers` pseudo-user.  When we use `--userns=auto`, it will pick automatically a range assigned to `containers`, so each container would have a sub range of the total allocation here.

I'd use at least something like 16M.  @rhatdan what do you think?

Reply:
Upstream [kubernetes sizing guide](https://kubernetes.io/docs/setup/best-practices/cluster-large/) suggests no more than around 110 containers per node.  Rounding up to a nice round 128, to assign 64k to each of them, we'd need 8388608, so I think 16M should be sufficient.

Reply:
yeah openshift supports 250, which would precisely fit in 16M :)

Reply:
updated to 16000000

Reply:
The containers in a pod share the same user namespace so the limitations is on the number of pods

Reply:
even better!